packages/mattermost-redux/src/constants/posts.ts

```diff
@@ -1,34 +1,36 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
+import {PostType} from 'mattermost-redux/types/posts';
+
 export const PostTypes = {
```

Comment:
Are we able to set the type here instead of having to cast each of these individually?
```suggestion
export const PostTypes: Record<string, PostType> = {
```

Reply:
Sort of. That's possible, but it makes `PostTypes.anything` defined as either a `PostType` or a `PostType | undefined`. Typing them individually is annoying, but it ensures only the actual values are defined.

Ideally, I think we'd scrap this whole structure and replace it with an enum, but that's a bigger task for later.